src/Components/Onboarding/SelectableToggle.tsx

```diff
@@ -11,34 +11,39 @@ interface SelectableToggleProps {
   selected: boolean
 }
 
-const IconContainer = styled.div`
-width: 15px;
-height: 15px;
-background-color: black;
-display: none;
-border-radius: 50%;
-float: right;
-margin-right: 15px;
+interface IconContainerProps extends React.HTMLProps<HTMLIFrameElement> {
+  isSelected: boolean
+}
+
+const iconContainer: StyledFunction<IconContainerProps> = styled.div
+
+const IconContainer = iconContainer`
+  width: 18px;
+  height: 18px;
+  background-color: black;
+  border-radius: 50%;
```

Comment:
I would expect the inverse circle check mark to be part of the icon font set. Or, the `Icon` component should display it using the `follow-circle.is-following` icon and the CSS here, as if the font set has it. Alternatively, you could implement `CircleIcon` or some sort.